dax/launcher.py

```diff
@@ -87,6 +87,9 @@ def __init__(self,
                  yaml_dict=dict(),
                  priority_project=None,
                  queue_limit='10',
+                 queue_limit_pending='10',
```

Comment:
should these defaults be ints?

Reply:
Didn't think about it - I found them this way. I can't quite tell - they get converted here maybe?
https://github.com/VUIIS/dax/blob/2dd32c65eae8e32b6b578b327df17fb552df5971/dax/dax_manager.py#L303

Reply:
...pretty sure they should be ints though so I changed 'em